pkg/lib/cockpit/_internal/transport.js

```diff
@@ -146,200 +138,210 @@ class Transport extends EventEmitter {
             this.close();
         };
 
-        this.#ws.onmessage = this.dispatch_data = (arg) => {
-            this.#got_message = true;
+        this.#ws.onmessage = event => this.dispatch_data(event);
```

Comment:
Sure! This should be more or less what .bind() does, except that looks rather ugly. This arrow is nicer.

Reply:
Ya.  I really dislike `.bind()`, `.apply()`, etc.  Modern language features give us better ways to write all of these things...